src/OrchardCore/OrchardCore.DisplayManagement/Notify/Notifier.cs

```diff
@@ -16,17 +17,21 @@ public Notifier(ILogger<Notifier> logger)
             _logger = logger;
         }
 
-        [Obsolete("This method will be removed in a later version. Use AddAsync()")]
-        // TODO The implementation for this is provided as an interface default implementation
-        // when the interface method is removed, replace this with AddAsync.
         public void Add(NotifyType type, LocalizedHtmlString message)
+        {
```

Comment:
Maybe we should just remove the `Add()` method and remove the base implementation of `AddAsync` from the interface.

Reply:
In a further release, the `[obsolete]` tag is new in this version I believe.

Reply:
@sebastienros it was added 3 years ago

https://github.com/OrchardCMS/OrchardCore/commit/1586fa0b18a41d6472116fd75e232543a53fe5a1#diff-31d363f31e73f51c9693507a04d4cb320a190c78d42c7a0dd597615a4edd6d37

Reply:
Great, I wanted to do more changes actually after using it. We can do that in another PR, I don't want to diverge too much.